### DISCUSSION TITLE
How can i catch the array length in truffle test.

### DISCUSSION THREAD

**Discussion by kh0alu** [ConsenSys-archive/truffle#5352] [General Q&A]
i know to get a value of one element in array we use instance.arrayName(index).then....
but how can i get the number of array length, please tell me how to do this. Thank you so much!

**Comment by leeftk** [comment]
Hey @kh0alu, thanks for submitting this. I believe your question is more related to JavaScript than Truffle itself but I'll gladly answer it for you here! When looking to get the length of an array you can use the `.length` method to get the result.

For example 

`const fruits = ["Banana", "Orange", "Apple", "Mango"];
let length = fruits.length;`

If we were to `console.log(length)` the result would be 4. Hope this helps. If I doesn't address your issue feel free to ask again here and we'll gladly help out however we can. Thanks!
